src/main.rs

```diff
@@ -1087,6 +1087,17 @@ fn main() {
                         binary with the Debug profile.")
                     .long("dbg-name")
                     .short("D"))
+                .arg(Arg::with_name("extension")
```

Comment:
To avoid confusion with WiX extensions, please change the argument name from `extension` to `file-extension`.